about.html

```diff
@@ -18,6 +18,7 @@ <h2 id="rules">Rules <a href="#rules">¶</a></h2>
 <li>Automated posts should be <strong>unlisted</strong> (rendering visible to everybody, but not appearing on the local timeline) to keep the local timeline of our instance a place of community dialogue and human interaction. This extends to bots, feed posters, Twitter "retweets" and Twitter crossposts with broken mentions.</li>
 <li>Bots may only interact with a user when they're invited by that user to do so.</li>
 <li>If you need to post with a high frequency, please set your posts to be <strong>unlisted</strong> to keep the local timeline readable.</li>
+<li>If you want to discuss a topic with others try to keep calm, use moderate laguage and try to avoid irony, sarcasm or similar types of communication that can lead to misunderstandings.</li>
```

Comment:
Hmm. While I like keeping calm, I don't like to suggest it to people this directly. I'd propose:

"In discussions, please remain civil, do not insult the people you're talking to, and try to avoid irony, sarcasm, or similar modes of languge – they typically don't translate well to written language tend to escalate discussions."

Reply:
Sounds good!